bundles/org.openhab.ui/web/src/components/widgets/system/oh-knob.vue

```diff
@@ -1,6 +1,16 @@
 <template>
-  <knob-control v-bind="config" :text-color="config.textColor || ($f7.data.themeOptions.dark === 'dark') ? '#ffffff' : undefined" :value="value"
-                @input="sendCommandDebounced($event)" @click.native.stop="sendCommandDebounced(value, true)" @touchend.native.stop="sendCommandDebounced(value, true)" />
+  <div :style="dottedPath">
+    <knob-control v-if="!config.sliderType" v-bind="config" :text-color="config.textColor || (($f7.data.themeOptions.dark === 'dark') ? '#ffffff' : undefined)" :value="value"
+                  @input="sendCommandDebounced($event)" @click.native.stop="sendCommandDebounced(value, true)" @touchend.native.stop="sendCommandDebounced(value, true)" />
+
+    <round-slider v-else :value="value" :min="config.min" :max="config.max" :step="config.stepSize" :radius="config.size/2"
```

Comment:
Using `v-bind` here would have been nice, but I see the reason why you did not: several paramaters have different names compared to the prop names.

Reply:
yes, exactly, that's why I couldn't use v-bind.